community/Hexo.gitignore

```diff
@@ -0,0 +1,19 @@
+# gitignore template for Hexo sites
+# website: https://hexo.io/
+# Recommended: Node.gitignore
+
+# Ignore generated files and directories
+.tmp*
+dist/
+public/
+coverage/
```

Comment:
I'm seeing reference to the `public/` directory in the Hexo docs, but not the `dist/` and `coverage/` directories 🤔 can you give a bit more context on what they are used for? I want to make sure we aren't excluding anything that may be useful to commit.

Reply:
Well I found them [here](https://github.com/hexojs/hexo/blob/master/.gitignore), maybe they're for the hexo project, not a hexo-built project, I think it's better to remove them.